web/services/models.py

```diff
@@ -73,15 +73,22 @@ def notify_report_received(sender, instance, signal, *args, **kwargs):
         # Don't send a notification if there was not email provided as we can't
         # actively do anything about it
         return
```

Comment:
Did we want to make the change that even if no email is provided we still get the slack notification with the stacktrace?